googleanalytics.pbs

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+##################
+####  PBS preamble
+
+#### #### ####  These are the most frequently changing options
+
+####  Job name
+#PBS -N googleanalytics
+
+####  Request resources here
+####    These are typically, number of processors, amount of memory,
+####    an the amount of time a job requires.  May include processor
+####    type, too.
+
+#PBS -l nodes=1:ppn=4,mem=32gb,walltime=1:00:00
+
+
+####  Flux account and queue specification here
+####    These will change if you work on multiple projects, or need
+####    special hardware, like large memory nodes or GPUs or,
+####    or if you use software that is restricted to campus use.
+
+#PBS -A engin_flux
+#PBS -q flux
+
+#### #### ####  These are the least frequently changing options
+
+####  Your e-mail address and when you want e-mail
+
+#PBS -M uniqname@umich.edu
```

Comment:
Use `${USER}@umich.edu` if that works

Reply:
I tried using `${USER}@umich.edu` but PBS seems to take that string literally, so it seems like we have to manually enter the email address